spec/facets/ExtractionErrorRunFacet.json

```diff
@@ -0,0 +1,51 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+	"$id": "https://openlineage.io/spec/facets/1-0-0/ExtractionErrorRunFacet.json",
+	"$defs": {
+		"ExtractionErrorRunFacet": {
+			"allOf": [{
+				"$ref": "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/RunFacet"
+			}, {
+				"type": "object",
+				"properties": {
+					"totalTasks": {
+						"description": "The number of distinguishable tasks that were processed by OpenLineage, whether successfully or not. Those could be, for example, distinct SQL statements.",
+						"type": "integer"
+					},
+					"failedTasks": {
+						"description": "The number of distinguishable tasks that were processed not successfully by OpenLineage. Those could be, for example, distinct SQL statements.",
+						"type": "integer"
+					},
+                    "errors": {
+                        "type": "array",
+                        "items": {
+                            "type" : "object",
+                            "properties" : {
+                                "errorMessage" : {
+                                    "description" : "Text representation of extraction error message.",
+                                    "type" : "string"
+                                },
+                                "task": {
+                                    "description": "Text representation of task that failed. This can be, for example, SQL statement that parser could not interpret.",
+                                    "type" : "string"
+                                },
+                                "taskNumber": {
+                                    "description": "Order of task (counted from 0).",
```

Comment:
Is this count of failed tasks or all? If latter, I don't think we have a concept of task in extractors to have them counted in any manner.

Reply:
In this case, this does not relate to Airflow tasks or extractors at all, but would be used as a count of "tasks" inside a single run - piece of work.

A good example is one SQL job that executes multiple statements, whether inside of outside of transaction.